coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -952,6 +952,30 @@ def instance_norm(context, node):
     )
     context.add(x)
 
+@register_torch_op
+def group_norm(context, node):
+    inputs = _get_inputs(context, node, expected=6)
+    x = inputs[0]
+    num_groups = inputs[1].val
+    weight = inputs[2]
+    bias = inputs[3]
+    eps = inputs[4]
+    n,c,h,w = x.shape[0],x.shape[1],x.shape[2],x.shape[3]
+    num_groups = min(num_groups,c)
+    x = mb.reshape(x=x, shape=[n,num_groups,c//num_groups,h,w])
+    mean = mb.reduce_mean(x=x, axes=[2,3,4], keep_dims=True)
```

Comment:
This is kind of inefficient as `reduce_mean` is also done as part of the `_std` operation, so the mean can be obtained from there directly, instead of adding another op for that.